src/D2L.CodeStyle.Analyzers/Async/Generator/AsyncToSyncMethodTransformer.cs

```diff
@@ -125,6 +125,15 @@ AttributeSyntax attribute
 			}
 		}
 
+		switch( returnTypeInfo.Type.MetadataName ) {

```

Comment:
Are both of these cases possible?  There is only one `IAsyncEnumerable<T>` from my understanding.  Can we check the containing namespace as well?

Reply:
The debugger just says global namespace, not System.Collections.Generic... does that sound right and is it specific enough to be checked?

Reply:
Nevermind, figured it out thanks!